cli/lib/exec/run.js

```diff
@@ -45,6 +45,11 @@ const processRunOptions = (options = {}) => {
 
   const args = ['--run-project', options.project]
 
+  // when false, send this to the server to ensure globally enabled auto cancellation is turned off
+  if (options.autoCancelAfterFailures || options.autoCancelAfterFailures === false) {
```

Comment:
Can the user disable auto cancellation by setting `autoCancelAfterFailures` to `0`?

Reply:
Yes! if the value is set to 0 it would override the project settings making it disable

Reply:
Make sure the above `if` statement gets updated since it will currently not pass. Probably want to add a test for it as well.